configbp/yaml.go

```diff
@@ -0,0 +1,33 @@
+package configbp
+
+import (
+	"fmt"
+	"strconv"
+
+	"gopkg.in/yaml.v2"
+)
+
+// Int64String is an int64 type that can be yaml deserialized from strings.
```

Comment:
🔕 (nit)

```suggestion
// Int64String is an int64 type that can be deserialized from yaml strings.
```

Reply:
I feel that "yaml string" doesn't really mean anything (or it means a string that's the yaml content, which is not what we meant here), while "yaml deserialize" could be used to mean the action of deserialize yaml content?